automation/taskcluster/lib/tasks.py

```diff
@@ -66,13 +67,13 @@ def build_task(self, name, description, command, dependencies=[],
         }
 
     def beetmover_task(self, name, description, version, artifact_id,
-                       dependencies=[], upstreamArtifacts=[], scopes=[]):
+                       dependencies=[], upstreamArtifacts=[], scopes=[], is_staging=False):
```

Comment:
Why does `build_task(...)` care about `is_staging` and `is_release`, but `beetmover_task` only cares about `is_staging`?
Related: 
* how do `is_staging` and `is_release` compare to each other
* what's the difference between `(is_staging: true, is_release: false)` and `(is_staging: true, is_release: true)`? Perhaps it would be better to have an enum for the states we care about?

Reply:
I thought `build_task()` was also used for PRs, but I just realized it's a release-only thing. I removed `is_release`, then